tests/write_font_test.py

```diff
@@ -43,60 +43,46 @@ def test_keep_glyph_names(svgs, color_format, keep_glyph_names):
         assert ufo.glyphOrder != ttfont.getGlyphOrder()
 
 
-@pytest.mark.parametrize("svgs", [("rect.svg", "one-o-clock.svg")])
 @pytest.mark.parametrize(
     "color_format",
     [
+        "glyf",
         "cff_colr_0",
-        "cff_colr_1",
-        "glyf_colr_0",
         "glyf_colr_1",
         "picosvg",
     ],
 )
-# Default version is 1.000
-def test_default_version(svgs, color_format):
-    config, glyph_inputs = test_helper.color_font_config(
-        {
-            "color_format": color_format,
-        },
-        svgs,
-    )
-    ufo, ttfont = write_font._generate_color_font(config, glyph_inputs)
-    ttfont = test_helper.reload_font(ttfont)
-
-    assert ufo.info.versionMajor == 1
-    assert ufo.info.versionMinor == 0
-
-
-@pytest.mark.parametrize("svgs", [("rect.svg", "one-o-clock.svg")])
 @pytest.mark.parametrize(
-    "color_format",
+    "version_major, version_minor, expected",
     [
-        "cff_colr_0",
-        "cff_colr_1",
-        "glyf_colr_0",
-        "glyf_colr_1",
-        "picosvg",
+        (None, None, "1.000"),  # default
+        (1, 2, "1.002"),
+        (None, 1, "1.001"),
+        (2, None, "2.000"),
+        (16, 28, "16.028"),
```

Comment:
@yisibl note how the version_minor gets left-padded with zeros, that's how OpenType versions are supposed to work. If you meant `16.280` you have to set version minor to `280` otherwise you get `16.028`. Does that make sense?

Reply:
I think we can consider adding `16.280` with three decimal places, and even a version number with four decimal places, which helps us understand what the final version number is.

Learned, very good example. Sorry, I just started to learn to use pytest.